src/js/server/review/logic.js

```diff
@@ -63,8 +64,29 @@ exports.getApplicationReview = function getApplicationReview(adminId, hackerAppl
 }
 
 exports.getNextApplicationToReviewForAdmin = function getNextApplicationToReviewForAdmin(admin) {
-  // TODO: Make this return something meaningful
-  return HackerApplication.findOne({
-    order: [ Sequelize.fn('RANDOM') ],
+  // To get an appropriate application we run the query stored at assign.sql
+  const assignmentQuery = fs.readFileSync('src/js/server/review/assign.sql', 'utf8');
```

Comment:
I know this has already passed review but this synchronous file read on every request is going to block every other request until it completes. Could we possibly move it outside of the function so the file is read once per application start?